src/platforms/common/data-management/debug-files/source-context.mdx

```diff
@@ -9,11 +9,23 @@ around the location of stack frames. Certain SDKs can resolve this source
 context automatically, such as the Python SDK, because they have access to
 unobfuscated source code at runtime.
 
-To get source context for native applications, source code needs to be uploaded
-alongside the debug information files. The recommended way to do this is by
-using `sentry-cli`. See [Creating Source Bundles](/product/cli/dif/#creating-source-bundles) for more information.
+<PlatformSection notSupported={["dotnet"]}>
 
-Source bundles show up as regular debug files on the _Debug Files_ settings
+To get source context for compiled applications, source code needs to be uploaded
+alongside the debug information files.  The recommended way to do this is by using `sentry-cli`.
+See [Creating Source Bundles](/product/cli/dif/#creating-source-bundles) for more information.
+
+</PlatformSection>
+
+<PlatformSection supported={["dotnet"]}>
+
+To get source context for compiled applications, source code needs to be uploaded
+alongside the debug information files.  For .NET projects, the recommended way to do this is
+through the built-in MSBuild properties, as described in the [MSBuild setup](../../../configuration/msbuild/) docs.
+
+</PlatformSection>
+
+Source code shows up as regular debug files on the _Debug Files_ settings
```

Comment:
Would be great to add an explicit path for users so that they can find this in their Sentry app. Something like: 

To access your "Debug Information Files" in Sentry, go to the [Projects] page, click on a project, click into settings, then navigate down to "Debug Files" in the page nav.